frontend/src/store/store.test.js

```diff
@@ -190,16 +217,19 @@ describe('Vuex store', () => {
         klickTipp: {
           newsletterState: true,
         },
+        gmsAllowed: true,
+        gmsPublishName: 'GMS_PUBLISH_NAME_FULL',
+        gmsPublishLocation: 'GMS_LOCATION_TYPE_EXACT',
         hasElopage: false,
         publisherId: 1234,
         roles: ['admin'],
         hideAmountGDD: false,
         hideAmountGDT: true,
       }
 
-      it('calls eleven commits', () => {
+      it('calls fifteen commits', () => {
```

Comment:
```suggestion
      it('calls fourteen commits', () => {
```
because the old code says `eleven commits` and in the new code are only three new commits added.

Reply:
schau mal was zwei Zeilen dadrunter steht, da steht 15.
Die Elf war schon nicht korrekt, wenn du vorher in den Code schaust gab es schon bereits 12 commit calls in login.

Reply:
ok